src/App.tsx

```diff
@@ -188,7 +192,8 @@ export function Layout() {
                 }
               >
                 <ViewTabs />
-                {/* <ReactQueryDevtools initialIsOpen={false} /> */}
+                <ReactQueryDevtools initialIsOpen={false} />
+                <ScrollRestoration />
```

Comment:
I don't think this is actually doing anything after all. I would remove it again if so.

Reply:
![image](https://github.com/user-attachments/assets/9ba80d3f-7099-4e43-a8c2-e7c601fbafce)

it add in the tanstack query dev tools. It was only remove before as it was causing the CI to fail

Reply:
your talking about the scrollRestoration arent you hahah sorry